modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache

```diff
@@ -161,14 +176,15 @@ public class ApiClient {
     JsonNullableModule jnm = new JsonNullableModule();
     mapper.registerModule(jnm);
     {{/openApiNullable}}
-    URI baseURI = URI.create("{{{basePath}}}");
-    scheme = baseURI.getScheme();
-    host = baseURI.getHost();
-    port = baseURI.getPort();
-    basePath = baseURI.getRawPath();
-    interceptor = null;
-    readTimeout = null;
-    responseInterceptor = null;
+    return mapper;
+  }
+
+  private void updateBaseUri(String baseUri) {
```

Comment:
Please make this `public`. It is useful. Updating the base URI can anyway be done today but requires calling 4 different setXXX methods (`setSchema`, `setPort`, ...).

Reply:
fixed